content/en/docs/languages/rust/libraries.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+description: How to instrument libraries an app depends on
+---
+
+{{% docs/languages/libraries-intro Rust %}}
+
+## Use Instrumentation Libraries
+
+Each instrumentation library is a [crate](https://crates.io/).
+
+For example,
+[the instrumentation library for Actix Web](https://crates.io/crates/actix-web-opentelemetry)
+will automatically create [spans](/docs/concepts/signals/traces/#spans) and
+[metrics](/docs/concepts/signals/metrics) based on the inbound HTTP requests.
+
+If your app or one of your dependencies is making use of the create
+[`tracing`](https://crates.io/crates/tracing) you can use
+[`tracing-opentelemetry`](https://crates.io/crates/tracing-opentelemetry) for
```

Comment:
```suggestion
[tracing-opentelemetry](https://crates.io/crates/tracing-opentelemetry) to
```

Reply:
tracing-opentelemetry is already listed in the registry as an instrumentation - don't think we need to specially call it out. (Agree it is very popular, but there are some interop issues, not fully sorted out..)

Reply:
@cijothomas I added this based on a comment from @hdost: https://github.com/open-telemetry/opentelemetry.io/pull/3828#discussion_r1460956728

Since you are the rust experts, I would leave it to you to have it or not

Reply:
No need to explicitly list, given it is already listed in the registry. It is not something maintained by the OpenTelemetry organization. (If this changes in the future, we can add it).

Reply:
let me remove it then once again

Reply:
If memory serves me, there's a ton of adoption of the tokio tracing crate and several folks in the Rust community recommend using that with the OTel exporter. Not a "now" thing but we should probably try to quantify that a bit, because if the community leans towards something we should also lean into it.

Reply:
Yes, it is an extremely popular crate. (we in OTel Rust, does some special handling for supporting that crate in our sdk :))

All I said was there is no need to explicitly recommend/call-out that, especially since there are unresolved issues with the combination of using tracing + otel. The otel rust maintainers and tokio-tracing maintainers are working to resolve them, and once the resolution is there, we can do an explicit call out. (it is unknown what that resolution would be... it could be that tracing-opentelemetry is merged into otel itself or vice-versa or something else. Whatever the resolution will be, it'll be first called out in OTel rust repo itself, and then we can move it to opentelemetry.io/docs too.)

Reply:
Awesome, thanks @cijothomas

Reply:
Let's go with @cijothomas' suggestion for now